pkg/controllers/management/auth/project_cluster_handler.go

```diff
@@ -442,6 +447,27 @@ func (m *mgr) reconcileResourceToNamespace(obj runtime.Object, controller string
 	})
 }
 
+func (m *mgr) createProjectAnnotation() (string, error) {
```

Comment:
A new function isn't needed, use `addRTAnnotation`. The project will just need to be created outside the call to `Create`, passed through `addRTAnnotation` then into create.

Reply:
sorry, I should've realized `addRTAnnotation` would work even before creating project, updated!